### ISSUE TITLE
Cannot increase or decrease number of replicas in a deployment via Rancher Dashboard without errors

### ISSUE THREAD

**Issue by adamkpickering** [rancher-sandbox/rancher-desktop#2095]
### Actual Behavior

When I click the buttons to increase or decrease the number of replicas in an existing deployment, there are red error boxes that pop up. I should be able to click the error boxes as many times as I want, and each click should register as another replica. This should be made more robust.

### Steps to Reproduce

1. Start Rancher Desktop. Once k8s is up and running, open Rancher Dashboard from the system tray.
2. Open Workload > Deployments and create a deployment if you don't already have one. Name "nginx" and image "nginx" works.
3. In the dropdown that appears when clicking the green bar under the `Health` column, click `+` or `-` a number of times rapidly. Red error boxes should pop up.

### Result

Cannot quickly increase or decrease number of replicas in a k8s deployment.

### Expected Behavior

I expected to be able to smoothly and reliably increase/decrease the number of replicas in a deployment.

### Rancher Desktop Version

7f2828a3

### Rancher Desktop K8s Version

1.22.7

### Which container runtime are you using?

containerd (nerdctl)

### What operating system are you using?

Other Linux

### Operating System / Build Version

Pop OS 20.04

### What CPU architecture are you using?

x64

### Linux only: what package format did you use to install Rancher Desktop?

N/A

**Comment by rak-phillip** [comment]
This might be fixed with #2883. We should test to validate after merging.

@jandubois

**Comment by jandubois** [comment]
> This might be fixed with #2883. We should test to validate after merging.

I can test this once it is merged. Ping me! 😄

**Comment by jandubois** [comment]
@rak-phillip I do see the red error box, so I assume this is not fixed:

<img width="1260" alt="Screen Shot 2022-09-14 at 3 01 08 PM" src="https://user-images.githubusercontent.com/78947/190270848-05ef218c-7994-4a37-a8d8-25c283b99170.png">

**Comment by rak-phillip** [comment]
@jandubois You're correct. Perhaps I was a little overly optimistic in my initial round of testing.. Thanks for taking the time to test 🙂

Upon further inspection, I was able to reproduce in multiple Rancher Manager environments outside of Rancher Desktop after taking a closer look.  

Since this behavior is present in other environments, I think it makes sense to close this issue after we've filed a new one in the appropriate repo. I'll follow up with more info.

**Comment by jandubois** [comment]
This is fixed in the update to Cluster Dashboard `2.11.0.rd1` which will be included in Rancher Desktop 1.18.
